### ISSUE TITLE
Subject pages should behave like /search?

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#2375]
This is related to and may be solved by #10760 and #10512

One of our patrons says:
> Is there a way to browse a subject that allows me to use, say, a list format (like a regular search page) rather than the carousel? For subjects that contain a lot of books--Diaries, for instance, has over 14K--looking at six books at a time is wildly inefficient, and there's no way to save your place and come back to it later. Worse, on my system (Windows 8.1; current Firefox), the carousel sticks every couple of clicks and I have to wait anywhere from a few seconds to several minutes for it to resolve itself and load the next section of carousel; after a while it refuses to load anymore at all. 
I really enjoy poking around in subjects just to see what's available, and I've looked for some setting or link to allow me to browse in list form, but--am I just overlooking it somewhere?

In the future we'd like to use carousels for the subject home page (https://openlibrary.org/subjects) which currently could use a lot of improvement, and perhaps instead show /search style view on the individual subjects pages.

**Comment by cdrini** [comment]
Give this to the person: https://openlibrary.org/search?q=subject%3ADiaries&mode=everything

**Comment by cdrini** [comment]
The search page also lets you filter/sort, which is nice :)

**Comment by tfmorris** [comment]
I agree that the carousel display paradigm is painful for this, but the search that feeds it is also broken / underperforming, which is what is causing the long delays that the user is seeing. In the example I just played with it took from 12 seconds for https://openlibrary.org/subjects/space_flight.json?limit=12&offset=12 to complete to a couple of seconds for most of the subsequent pages (which is still pretty long such a small request).

Separately, we should investigate why these search queries are so slow.

**Comment by seabelis** [comment]
Yes, please. The ability to filter on additional subjects within results would also be handy.

**Comment by guyjeangilles** [comment]
This seems like a combination of problems: 1) The subject page needs a redesign 2) Subject search is broken/performing poorly. Should 2) be it's own issue?

**Comment by tfmorris** [comment]
There no longer seems to be an issue with the search performance (the example above completes in ~250 msec even when varying the offset), so I suggest keeping this focused on the UI aspect.

Perhaps its as simple as offering a UI affordance which allows the user to choose https://openlibrary.org/search?q=subject%3ADiaries&mode=everything and an alternative to https://openlibrary.org/subjects/diaries which is a 14,000+ entry carousel.

**Comment by tfmorris** [comment]
Changing the subject page layout is going to expose some data quality issues. The 1st and 3rd of the top "diary" authors are publishers of wall calendars.

[Abrams](https://openlibrary.org/search?q=subject%3ADiaries&author_key=OL2626615A&mode=everything)
[Wall](https://openlibrary.org/search?q=subject%3ADiaries&author_key=OL2627359A&mode=everything)

And other top "authors" include publishers of both blank diaries/journals and wall calendars:

https://openlibrary.org/search?q=subject%3ADiaries&author_key=OL2879178A&mode=everything
https://openlibrary.org/search?q=subject%3ADiaries&author_key=OL3833474A&mode=everything
[Blank Journals](https://openlibrary.org/search?q=subject%3ADiaries&author_key=OL3015549A&mode=everything)
[Paperblank Book Company](https://openlibrary.org/search?q=subject%3ADiaries&author_key=OL3015547A&mode=everything)

**Comment by LeadSongDog** [comment]
Excellent! Hidden problems don't get fixed.

**Comment by mekarpeles** [comment]
This is in-line with #8279

**Comment by mekarpeles** [comment]
Closing in favor of #10760 and #10512
